libservice/headers/service/Service.h

```diff
@@ -29,9 +44,14 @@ struct Service {
 	uint32_t internalClientsNumber{0u};
 	uint32_t externalClientsNumber{0u};
 
+	std::unordered_map<std::array<uint8_t, 2>, std::chrono::time_point<std::chrono::steady_clock>, ArrayHasher> detectedExternalIPv416Networks;
```

Comment:
types std::array<uint8_t, 3> and std::array<uint8_t, 2> are strange and not efficient why simply not uitnt32_t

Reply:
I wanted to be consistent in my solution. I'll change that.